adapters/InMobi/InMobiAdapter/GADMAdapterInMobiInterstitialAd.h

```diff
@@ -0,0 +1,19 @@
+//
+//  GADMAdapterInMobiInterstitialAd.h
+//  IMAdMobAdapter
+//
+//  Created by Bavirisetti.Dinesh on 02/09/22.
+//
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+#import <InMobiSDK/IMInterstitial.h>
+
+@interface GADMAdapterInMobiInterstitialAd : NSObject<GADMediationInterstitialAd, IMInterstitialDelegate>
+
+/// Initializes the Interstitial ad renderer.
+- (nonnull instancetype)initWithPlacementIdentifier:(nonnull NSNumber *)placementIdentifier;
+
```

Comment:
Ditto, don't think this is needed, can remove. The same applies to all ad formats.